bin/templates/cordova/Api.js

```diff
@@ -112,16 +110,13 @@ Api.createPlatform = function (destination, config, options, events) {
     events = setupEvents(events);
     var result;
     try {
-        result = require('../../lib/create')
-        .create(destination, config, options, events)
-        .then(function (destination) {
+        result = require('../../lib/create').create(destination, config, options, events).then(function (destination) {
```

Comment:
This is an unfortunate workaround for eslint-ing.  I would much prefer this be broken up, ala ...

    var libCreate = require('../../lib/create');
    var result = libCreate.create(destination, config, options, events);
    result.then(function (destination) {
        // ...
    });
    ...
    return result;